runtime/v1/linux/proc/exec.go

```diff
@@ -144,7 +145,7 @@ func (e *execProcess) Kill(ctx context.Context, sig uint32, _ bool) error {
 
 func (e *execProcess) kill(ctx context.Context, sig uint32, _ bool) error {
 	pid := e.pid
-	if pid != 0 {
+	if pid > -1 {
```

Comment:
\> 0?

Reply:
Yes, a not only small, but also big error. I'll pushed again.